src/libraries/System.Numerics.Vectors/ref/System.Numerics.Vectors.cs

```diff
@@ -7,6 +7,118 @@
 
 namespace System.Numerics
 {
+    public readonly struct Vector2<T> : IEquatable<Vector2<T>>, IFormattable
```

Comment:
This is going to need to be autogenerated via the ref assembly generator. Won't block on this though.